microshift_release_notes/microshift-4-18-release-notes.adoc

```diff
@@ -162,3 +162,23 @@ See the latest images included with {microshift-short} by xref:../microshift_upd
 ==== Bug fix
 
 * Previously, a failed greenboot health check flag did not clear when an RPM-install-based {microshift-short} service was updated without rebooting the system. As a consequence, greenboot health checks for optional components continue to fail because the `systemctl restart greenboot-healthcheck.service` command fails while the flag is present. With this release, the primary {microshift-short} greenboot health check clears the condition that causes optional component health checks to continue to fail. (link:https://issues.redhat.com/browse/OCPBUGS-51198[*OCPBUGS-51198*])
+
+[id="microshift-4-18-11-dp_{context}"]
+=== RHBA-2025:4249 - {microshift-short} 4.18.11 bug fix and enhancement advisory
+
+Issued: 1 May 2025
+
+{product-title} release 4.18.11 is now available, see the link:https://access.redhat.com/errata/RHBA-2025:4249[RHBA-2025:4249] advisory. Release notes for bug fixes and enhancements are provided in this documentation. The images that are included in the update are provided by the {OCP} link:https://access.redhat.com/errata/RHSA-2025:4211[RHSA-2025:4211] advisory.
+
+See the latest images included with {microshift-short} by xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[listing the contents of the {microshift-short} RPM release package].
+
+[id="microshift-4-18-11-bugs_{context}"]
+==== Bug fixes
+
+//* Previously, sos reports were missing firewalld information. With this release, the {microshift-short} sos report includes firewalld information for enhanced troubleshooting. (link:https://issues.redhat.com/browse/OCPBUGS-55002[*OCPBUGS-55002*])
```

Comment:
Just FYI, I noticed during peer-review that @dfitzmau has moved away from the *OCPBUGS* for core.